### ISSUE TITLE
Solute Adjusters should be "horizontal" not "vertical"

### ISSUE THREAD

**Issue by terracoda** [phetsims/membrane-transport#119]
Please update the orientation of the Solute Amount Adjusters to be horizontal.
![Image](https://github.com/user-attachments/assets/5676d902-40c8-4520-bd1d-d9578860896f)

@samreid, re-assign to @jessegreenberg if this is not something you know how to do yet.

**Question:**  Is the orientation attribute readily available as an option in the API? I feel, this is something devs should know to do on sliders and spinners when they see the design. I don't think designers should have to explicitly say this in the design doc, unless it is not obvious like in John Travoltage.

**Comment by samreid** [comment]
I saw that FineCoarseNumberSpinners are always horizontal, so I specified the default ariaOrientation there. Seems OK in my testing. @terracoda can you please verify?

**Comment by terracoda** [comment]
That's great to set the default to horizontal if all FineCoarseNumberSpinners are always horizontal.

**Comment by terracoda** [comment]
For horizontal orientation  is not being read out which is fine. It's only an issue if the incorrect orientation is read out. I think horizontal might be the default orientation in aria spec (not sure though), so that might be the reason for the orientation silence.

Issue is ready to close.
